.hound.yml

```diff
@@ -0,0 +1,4 @@
+scss:
+  enabled: false

```

Comment:
Did you notice that `scss` style check has been disabled here. Also add the config file used by scss-lint.